presto-main/src/main/java/com/facebook/presto/server/ServerConfig.java

```diff
@@ -39,6 +39,7 @@ public class ServerConfig
     private boolean queryResultsCompressionEnabled = true;
     private NodePoolType poolType = DEFAULT;
     private Duration clusterStatsExpirationDuration = new Duration(0, MILLISECONDS);
+    private boolean nestedDataSerializationEnabled = true;
```

Comment:
Why is the default true ? Like why go against backward compatibility ?

Reply:
We got consensus in #20588 that given this is a bug fix, the default should be to fix it, but allow a way to back out of it in case any reliance on this unfortunate behavior has been developed. Release notes will indicate to users the change in behavior, which is expected as any system evolves.